post_regression.py

```diff
@@ -0,0 +1,140 @@
+import model
+import numpy as np
+from sklearn.utils import resample
+
+def ci_bootstrap(model, t_obs, n_I_obs, population, alpha=0.95, n_iter=1000, r0_ci = True):
```

Comment:
Should we move this to a [Mixin](https://easyaspython.com/mixins-for-fun-and-profit-cb9962760556) now or in a follow up?

Reply:
This function and the one below have the same signature. I suggest to add a class member that allows to choose between different methods.
E.g `my_sir.ci(method="bootstrap", t_obs, ...)

Reply:
I agree. I think it is a good idea but for now they have different inputs and outputs. I updated that. Let's think on a better structure for the next PR